### PR TITLE
fix(SUP-14173): Change screen orientation detection

### PR DESCRIPTION
**Pull request by ErezBouskila**
@eitanavgil Please review PR.
Thanks

### CODE REVIEW

File: modules/Quiz/resources/quiz.js [hunk 1]
```diff
@@ -689,7 +689,7 @@
         isPortrait: function () {
             if (
                 this.getPlayer().getInterface().hasClass("mobile")
-                && (screen.orientation.angle === 0 || screen.orientation.angle === 180)
```
Comment (eitanavgil) [inline modules/Quiz/resources/quiz.js]: @ErezBouskila  
This looks OK but I am afraid that at some point one of the element in the chain parent.document.body.parentNode.clientWidth or parent.document.body.parentNode.clientHeight will be undefined. Perhaps in some embed types like iframe) 
Can we add a safer test ( if parent && parent.document .... )

**Comment by eitanavgil** [comment]
@OrenMe 
@ErezBouskila  said he tested iframes in KMC and also outside - this is why I asked to add this over-protecting tests. U think we need to also add a try& catch?

**Comment by OrenMe** [comment]
@eitanavgil there’s a method to check if you are in a friendly iframe or not. It should suffice. No need for try catch or lengthy checks.

**Comment by ErezBouskila** [comment]
@eitanavgil @OrenMe Please review new fix.
Thanks

**Pull request merged**